README.md

```diff
@@ -2,18 +2,318 @@
 
 # Kimchi
 
-**K** - _Kotlin_  
-**i** - _Inject_  
-**m** - _Merge_  
-**c** - _Component_  
-**h** - _Hints_  
-**i** - _Intuitively_  
+**K**_otlin_ **I**_nject_ **M**_erge_ **C**_omponent_ **H**_ints_ **I**_ntuitively_
 
-Kimchi is an [Anvil](https://github.com/square/anvil)-like KSP processor for [kotlin-inject](https://github.com/evant/kotlin-inject) that lets you contribute bindings, modules, and subcomponents across multi-module projects to form your dependency injection graph without manual wiring.
+Kimchi is an [Anvil](https://github.com/square/anvil)-like KSP processor for [kotlin-inject](https://github.com/evant/kotlin-inject) that lets you contribute bindings, modules, and subcomponents across multi-module projects to form your dependency injection graph without having manual wiring your components upstream.
 
 ## Getting started
 
-🚧 _THIS LIBRARY IS A WIP_ 🚧
+Getting started is easy! Just create a component abstract class or interface annotated with `@MergeComponent` like so
+
+```kotlin
+@MergeComponent(AppScope::class)
+abstract class AppComponent
+```
+
+Now we have a component ready to collect contributions on the scope `AppScope`
+Next, start contributing modules and bindings.
+
+### Binding
+```kotlin
+interface MenuRepository {
+  suspend fun getItems(): List<MenuItem>
+}
+
+@ContributesBinding(AppScope::class)
+@Inject
+class MenuRepositoryImpl(/*…*/) : MenuRepository {
+  override suspend fun getItems(): List<MenuItem> = //…
+}
+```
+
+### Multi-bindings
+```kotlin
+interface AppInitializer // { … }
+
+@ContributesMultibinding(AppScope::class)
+@Inject
+class AnalyticsInitializer() : AppInitializer
+
+@ContributesMultibinding(AppScope::class)
+@Inject
+class LoggingInitializer() : AppInitializer
+
+// Will come together in a Set
+@MergeComponent(AppScope::class)
+abstract class AppComponent {
+  abstract val initializers: Set<AppInitializer>
+```
+or you can specify a `MapKey` (`StringKey`, `IntKey`, `LongKey`, or you can make a custom key if needed) for mapped bindings
```

Comment:
this is clunky, but not sure how to improve. maybe something like

or you can specify a `MapKey` (`StringKey`, `IntKey`, `LongKey`, or custom key) for mapped bindings